### ISSUE TITLE
[Unhandled promise rejection: Error: Could not initialize Braintree SDK]

### ISSUE THREAD

**Issue by greenbaypayment** [smarkets/react-native-paypal#15]
I ejected my project from expo.

does this library support expokit project?

if anybody know the way to resolve this issue, help me

Thanks

**Comment by RocKhalil** [comment]
@greenbaypayment did you find the issue ?

**Comment by littletower** [comment]
@greenbaypayment, I'm getting this error as well, did you ever manage to solve it? thanks
(I'm on RN > 60)

**Comment by Mavenic** [comment]
Having the same issue?

**Comment by nzankich** [comment]
@greenbaypayment sorry for the slow reply

If I had to guess, this is because of an incorrect paypal token.  The braintree sdk is _very_ picky about what is sent in.  For example, if you send in extra spaces you'll get the error you posted (in that case be sure to use `.trim()`!)

If you look inside of the xCode logs you should see more detailed error logging

I can reproduce your issue by sending in "foo" instead of my paypal token
```    
const { nonce, payerId } = yield call(setupExecutePaypal, {
      amount,
      currency,
      locale,
      paypalToken: 'foo',
    })
```
In the react native logs I get the error you posted `Error: Could not initialize Braintree SDK]` but in the xCode logs I see this `[BraintreeSDK] ERROR BTClient could not initialize because the provided authorization was invalid`

Could you please double check your token and paste the xCode error logs here as well?

**Comment by nzankich** [comment]
you can test with this token `sandbox_9dbg82cq_dcpspy2brwdjr3qn`

**Comment by ytt123** [comment]
> you can test with this token `sandbox_9dbg82cq_dcpspy2brwdjr3qn`
how to get the  correct token on earch? I try your token  can open the another paypal page，I think it is right

**Comment by FarKotw** [comment]
For me for the time when I click payment button, I am getting this error.

I noticed the following warning in the logs. Unlinked the manual link and for the first time itself the button working without error. 
error React Native CLI uses autolinking for native dependencies, but the following modules are linked manually: 
  - react-native-paypal (to unlink run: "react-native unlink react-native-paypal

**Comment by nzankich** [comment]
Hey everyone, I've updated the README and created a new example app included in the new version of the package (3.1.0).  Please check that out and see if it helps diagnose and fix this bug.  Message me here if you are still experiencing issues and I will reopen

**Comment by NikK4999** [comment]
braintreeClient = BTAPIClient(authorization: PAYPAL_PAY.CLIENT_ID_LIVE)
 
why it is not working with the clinet_id.
